packages/wp-now/src/wp-now.ts

```diff
@@ -42,20 +43,23 @@ export default async function startWPNow(
 	options: Partial<WPNowOptions> = {}
 ): Promise<{ php: NodePHP; phpInstances: NodePHP[]; options: WPNowOptions }> {
 	const { documentRoot } = options;
-	const nodePHPOptions: PHPLoaderOptions = {
-		requestHandler: {
-			documentRoot,
-			absoluteUrl: options.absoluteUrl,
-		},
+	const nodePHPOptions: PHPRequestHandlerConfiguration = {
+		documentRoot,
+		absoluteUrl: options.absoluteUrl,
 	};
 
 	const phpInstances = [];
+	const loadRuntime = async () =>
+		await NodePHP.loadRuntime(options.phpVersion);
 	for (let i = 0; i < Math.max(options.numberOfPhpInstances, 1); i++) {
-		phpInstances.push(
-			await NodePHP.load(options.phpVersion, nodePHPOptions)
-		);
+		phpInstances.push(new NodePHP(await loadRuntime(), nodePHPOptions));
 	}
 	const php = phpInstances[0];
+	await rotatePHPRuntime({
+		php,
+		recreateRuntime: loadRuntime,
+		maxRequests: 400,
```

Comment:
Maybe let's try `100`? Also, it would be nice to detect out of memory errors and auto-rotate – not that it's a blocker here.

```suggestion
		maxRequests: 100,
```